extensions/oauth/src/test/resources/operations/operations-config.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:oauth="http://www.mulesoft.org/schema/mule/oauth"
+      xsi:schemaLocation="
+       http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+       http://www.mulesoft.org/schema/mule/oauth http://www.mulesoft.org/schema/mule/oauth/current/mule-oauth.xsd">
+
+    <flow name="accessTokenFlow">
+        <oauth:access-token tokenManager="tokenManagerConfig" resourceOwnerId="#[variables.resourceOwnerId]"/>
+    </flow>
+
+    <flow name="refreshTokenFlow">
+        <oauth:refresh-token tokenManager="tokenManagerConfig" resourceOwnerId="#[variables.resourceOwnerId]"/>
```

Comment:
We need to find a better name. Something like "retrieve-refresh-token", if not it would seem like this operation is doing a "refresh token". Once we decide the name for this operation, it should be apply to everything else (retrieve-access-token, retrieve-expires-in, etc)